pom.xml

```diff
@@ -6,7 +6,7 @@
   <!-- Arifact name and version information -->
   <groupId>net.snowflake</groupId>
   <artifactId>snowflake-ingest-sdk</artifactId>
-  <version>2.0.1</version>
+  <version>2.0.2-SNAPSHOT</version>
```

Comment:
Do we need your other [PR](https://github.com/snowflakedb/snowflake-ingest-java/pull/557) as well?

Reply:
Also confused about the 2 PRs. 
Should this just be 2.0.2 ?

Snapshot release should be 2.0.3-SNAPSHOT

Reply:
We're releasing on https://github.com/snowflakedb/snowflake-ingest-java/tree/tzhang-si-ga, so I'm porting every change that needs to be included manually one by one, the change comes from https://github.com/snowflakedb/snowflake-ingest-java/commit/e9103dd955061a12f0ce5414b0e138235d30a8b0, I will cherry-pick another which will override this to 2.0.2, see https://github.com/snowflakedb/snowflake-ingest-java/commit/79b4ea26e359046c4e347f71135e03879ff69923